RSA/simple2.c

```diff
@@ -0,0 +1,192 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <math.h>
+
+//функция, котоая проверяет число на простоту
```

Comment:
Fix typo pls.

Moreover, it looks like you should write all comments in English. Anyway, I've seen only English comments in main branch.